workflow-templates/docker-build-push.yaml

```diff
@@ -0,0 +1,64 @@
+#  ____   _                            _
+# / ___| | |__    __ _  _ __  ___   __| |
+# \___ \ | '_ \  / _` || '__|/ _ \ / _` |
+#  ___) || | | || (_| || |  |  __/| (_| |
+# |____/ |_| |_| \__,_||_|   \___| \__,_|
+# __        __            _      __  _
+# \ \      / /___   _ __ | | __ / _|| |  ___ __      __ ___
+#  \ \ /\ / // _ \ | '__|| |/ /| |_ | | / _ \\ \ /\ / // __|
+#   \ V  V /| (_) || |   |   < |  _|| || (_) |\ V  V / \__ \
+#    \_/\_/  \___/ |_|   |_|\_\|_|  |_| \___/  \_/\_/  |___/
+#        ____                _
+# __/\__|  _ \   ___    ___ | | __ ___  _ __
+# \    /| | | | / _ \  / __|| |/ // _ \| '__|
+# /_  _\| |_| || (_) || (__ |   <|  __/| |
+#   \/  |____/  \___/  \___||_|\_\\___||_|
+#  _             _  _      _   ___    ____               _
+# | |__   _   _ (_)| |  __| | ( _ )  |  _ \  _   _  ___ | |__
+# | '_ \ | | | || || | / _` | / _ \/\| |_) || | | |/ __|| '_ \
+# | |_) || |_| || || || (_| || (_>  <|  __/ | |_| |\__ \| | | |
+# |_.__/  \__,_||_||_| \__,_| \___/\/|_|     \__,_||___/|_| |_|
+#
+# Shared Workflows - Docker Build and Push
+#
+# Reusable Workflows used:
+# * Docker Lint
+#   * Readme <https://github.com/entur/gha-docker/blob/main/README-lint.md>
+#   * Owner: Team Plattform
+#   * Slack: #Talk-plattform
+# * Docker Build
+#   * Readme <https://github.com/entur/gha-docker/blob/main/README-build.md>
+#   * Owner: Team Plattform
+#   * Slack: #Talk-plattform
+# * Docker Push
+#   * Readme: <https://github.com/entur/gha-docker/blob/main/README-push.md>
+#   * Owner: Team Plattform
+#   * Slack: #Talk-plattform
+# Prerequisites:
+#    Appfactory Setup
+# TODO: This file comes from entur/.github/workflows/ci-docker.yaml
+#       Simply delete what you do not need. This file is yours now.
+#
+name: Docker Build and Push
+on:
+  pull_request:
+    branches:
+      - $default-branch
+jobs:
+  docker-lint:
+    name: Docker Lint
+    uses: entur/gha-docker/.github/workflows/lint.yml@v1
+    with:
+      dockerfile: Dockerfile
+  docker-build:
+    name: Build Docker Image
+    needs: docker-lint
+    uses: entur/gha-docker/.github/workflows/build.yml@v1
+    with:
+      dockerfile: Dockerfile
+  docker-push:
+    name: Push Docker Image
+    needs: docker-build
+    uses: entur/gha-docker/.github/workflows/push.yml@v1
+    with:
+      environment: prd
```

Comment:
prd is the default in the reusable workflow, and is no longer related to the github environment. Can be skipped